src/cmake/Modules/Platform/Wack.cmake

```diff
@@ -0,0 +1,29 @@
+# WACK (WebAssembly Clang Kit) is an experimental WebAssembly standalone
```

Comment:
Why do we need two Wack.cmakes that aren't the same?

Reply:
This one is the "platform file" which is in a magic special location in a path in the CMAKE_MODULE_PATH set up by the other file. It gets automatically executed when you set `CMAKE_SYSTEM_NAME` in the main file. I was just going to put all the content in here, but I found that the stuff in the other file had to go before this file is invoked, because it affects other CMake-provided files that get executed before this file does.